common/src/main/resources/i18n/translations_zh.properties

```diff
@@ -21,6 +21,7 @@ error_element_for_document_type_not_found=找不到文档类型为“{1}”且uu
 error_binaryfield_not_found_with_name=找不到二进制字段“{0}”。
 error_field_not_found_with_name=找不到字段“{0}”。
 error_found_field_is_not_binary=字段“{0}”的数据模型不是二进制字段模型。
+error_found_field_is_not_s3_binary=字段“{0}”的数据模型不是二进制字段模型。
```

Comment:
I see the value being copied from plain binary message. Since no one of us speaks Chinese, I would suggest getting the English version through translator service, and paste the results here (until a native Chinese speaker proposes a better translation).

Reply:
You can check this again.